test/jdk/javax/swing/JFrame/JFrameBackgroundRefreshTest.java

```diff
@@ -136,17 +137,23 @@ private static void moveMouseSlowly( int targetX, int targetY) {
     }
 
     private static boolean compareImages(BufferedImage bufferedImage) {
-        int sampleRGB = bufferedImage.getRGB(0,0);
+        Color firstPixel = new Color(bufferedImage.getRGB(0,0));
         for (int x = 0; x < bufferedImage.getWidth(); x++) {
             for (int y = 0; y < bufferedImage.getHeight(); y++) {
-                if (bufferedImage.getRGB(x, y) != sampleRGB) {
+                if (!compareColor(firstPixel, new Color(bufferedImage.getRGB(x, y)))) {
                     return false;
                 }
             }
         }
         return true;
     }
 
+    private static boolean compareColor(Color c1, Color c2) {
+        return Math.abs(c1.getRed() - c2.getRed()) < TOLERANCE &&
+                Math.abs(c1.getGreen() - c2.getGreen()) < TOLERANCE &&
+                Math.abs(c1.getBlue() - c2.getBlue()) < TOLERANCE;
+    }
```

Comment:
```suggestion
    private static boolean compareColor(Color c1, Color c2) {
        return Math.abs(c1.getRed() - c2.getRed()) < TOLERANCE
                && Math.abs(c1.getGreen() - c2.getGreen()) < TOLERANCE
                && Math.abs(c1.getBlue() - c2.getBlue()) < TOLERANCE;
    }
```
Java Coding Style recommends putting operators on the continuation line, it makes it obvious to the reader it is a continuation of an expression.